docs/source/data/bentoml-setup.yaml

```diff
@@ -0,0 +1,14 @@
+name: ci
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bentoml/setup-bentoml-action@v1
+        with:
+          python-version: '3.10'
+          bentoml-version: 'main'
+          cache: 'pip'
```

Comment:
I dont' think this work yet. Let me update the implementation on setup-bentoml-action. Let's just remove this for now.
```suggestion
```